photon-core/src/main/java/org/photonvision/vision/pipeline/AprilTagPipeline.java

```diff
@@ -134,8 +142,13 @@ protected CVPipelineResult process(Frame frame, AprilTagPipelineSettings setting
             return new CVPipelineResult(frame.sequenceID, 0, 0, List.of(), frame);
         }
 
+        CVPipeResult<CVMat> croppedFrame = cropPipe.run(frame.processedImage);
+        
+        sumPipeNanosElapsed += croppedFrame.nanosElapsed;
+
         CVPipeResult<List<AprilTagDetection>> tagDetectionPipeResult;
-        tagDetectionPipeResult = aprilTagDetectionPipe.run(frame.processedImage);
+        tagDetectionPipeResult = aprilTagDetectionPipe.run(croppedFrame.output);
+        
```

Comment:
Basically right here, make an UncropApriltagsPipe class that offsets all tag corners by staticCrop in x and y. And don't change the TargetCalculationParameters below.